config/prow/plugins.yaml

```
@@ -102,6 +102,14 @@ approve:
   - kubernetes-sigs/gateway-api
   lgtm_acts_as_approve: false
 
+help:
+  help_guidelines_summary: |
+                        Please ensure that the issue body includes answers to the following questions:
+                        - Why are we solving this issue?
+                        - If any, what changes need to be made to the code and what places can the assignee treat as reference points?
```

I think folks who have issues with reading comprehension in English, such as folks who don't know English as their first language or people with dyslexia, might have problems with the sentence structure here. Here's a suggestion:

```suggestion
                        - To address this issue, are there any code changes? If there are code changes, what needs to be done in the code and what places can the assignee treat as reference points?
```

---

This is excellent, thanks a bunch @nimbinatus :)

---

Done!